includes/multi-currency/MultiCurrency.php

```diff
@@ -1326,6 +1326,31 @@ public function is_initialized(): bool {
 		return static::$is_initialized;
 	}
 
+	/**
+	 * Returns the amount with the backend format.
+	 *
+	 * @param float $amount The amount to format.
+	 * @param array $args The arguments to pass to wc_price.
+	 *
+	 * @return string The formatted amount.
+	 */
+	public function get_backend_formatted_wc_price( float $amount, array $args = [] ): string {
+		if ( ! self::has_additional_currencies_enabled() ) {
```

Comment:
To be safe, I think we also need to check `WC_Payments_Features::is_customer_multi_currency_enabled()`.

Reply:
Added a check for both.